### ISSUE TITLE
Active development

### ISSUE THREAD

**Issue by smargold476** [turtl/tracker#426]
Is there any active development?

**Comment by skaag** [comment]
I highly doubt it... The last changes/updates are either 3 years ago or 5 years ago...

I came here to check for any hints on the export file format since I'm moving my data to Psono (check it out), and saw your comment 🤣 (which I did not expect since this project seems dead).

**Comment by smargold476** [comment]
I moved to ente.io as it is really nice and self hosting was straight forward ;)

**Comment by Go-rom** [comment]
Something happened two months ago on the Core repository : https://github.com/turtl/core/commits/master/

 Any news @orthecreedence  please ?

**Comment by orthecreedence** [comment]
Yeah, slowly working on the next version. This is going to be a p2p version of Turtl, no central servers required (although relay nodes will be useful for syncing data when your devices aren't all connected). The working idea is currently to have an interface somewhat similar to Notion, but with all the usual privacy guarantees of Turtl.

The idea is notes are made up of editable sections of text/data. Notes can link between each other and will be autosaved and versioned. And yes, it'll have dark mode.

This is a pretty immense undertaking and will likely take a while to get off the ground. Most of the encryption/syncing/relay stuff is going to be handled by a project ([Stamp protocol](https://stampid.net)) I'm putting a lot of time into right now so working on foundational stuff at the moment.

The goal is to be able to compile the Stamp/Turtl core to WASM so people can work on the UI without having to have an entire Rust toolchain handy.

Wish me luck =]

**Comment by Go-rom** [comment]
Thank you for your answer @orthecreedence and good luck with this new version of Turtl !
